net_transport.go

```diff
@@ -111,6 +111,11 @@ type StreamLayer interface {
 	Dial(address ServerAddress, timeout time.Duration) (net.Conn, error)
 }
 
```

Comment:
Can you also add a test case to net_transport_test.go that sets the EstimatedSize param instead of size?